apps/src/code-studio/pd/professional_learning_landing/landingPage.jsx

```diff
@@ -66,11 +66,12 @@ const LandingPage = React.createClass({
       return (
         <div>
           <h3>
-            Thank you
+            Submit your feedback!
```

Comment:
Any reason for the discrepancy in the heading between CSF and other (different wording: "Complete Survey" vs. "Submit your feedback" and different capitalization styles)?